Source/Authentication/ClaimHasMultipleValues.cs

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+using System;
+
+namespace Dolittle.AspNetCore.Authentication
+{
+
+    /// <summary>
+    /// The exception that gets thrown when there is a claim header on the http request with multiple values
+    /// </summary>
+    [Serializable]
```

Comment:
Suggest removing this attribute; you don't need to make it serializable - this is something inherited from the desktop framework to satisfy the internal serializers and specifically the binary serializers when doing marshalling between different app domains